spring-integration-ws/src/main/java/org/springframework/integration/ws/dsl/MarshallingWsOutboundGatewaySpec.java

```diff
@@ -63,9 +66,9 @@ public static class MarshallingWsOutboundGatewayNoTemplateSpec
 			extends BaseWsOutboundGatewaySpec<MarshallingWsOutboundGatewayNoTemplateSpec,
 			MarshallingWebServiceOutboundGateway> {
 
-		protected Marshaller gatewayMarshaller; // NOSONAR
+		protected Marshaller gatewayMarshaller = new Jaxb2Marshaller(); // NOSONAR
```

Comment:
Therefore we can remove that noisy `// NOSONAR`.

Honestly we discontinued to use SonarQube in our projects.
So, all those comments are redundant from now on.

Thanks